starter_code/views/partials/beerPartial.hbs

```diff
@@ -0,0 +1,7 @@
+
+<div class="beers">
+    <img class="beers-image" src="{{image_url}}" alt="">
+    <h5 class="beer-name">{{name}}</h5>
+     <h6 class="tagline">{{tagline}}</h6>
+     <p class="description">{{description}}</p>
+</div>
```

Comment:
et que tu rappelles this dans ton partial : 
  <h5 class="beer-name">{{this.name}}</h5>
     <h6 class="tagline">{{this.tagline}}</h6>
     <p class="description">{{this.description}}</p>